index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Material+Icons+Outlined"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Material+Icons+Round"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Material+Icons+Sharp"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Material+Icons+Two+Tone"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="main">
+      <header>
+        <nav class="mobile-version">
+          <span
+            class="nav-icon material-icons menu" id="menu"
+          >
+            menu
+          </span>
+        </nav>
+        <div class="logo">
+          <img src="./Img/Svg/sony-logo-3066.png" alt="" />
+        </div>
+        <nav class="desktop-version" id="show">
+          <ul class="nav-bar">
+            <span
+              class="visible close" id="close"
+            >
+              <img src="./Img/Cl/close_FILL0_wght400_GRAD0_opsz48.png" alt="close" />
+            </span>
+            <li> <a class="navbar-fun" href="./about.html">About</a></li>
+            <li> <a class="navbar-fun" href="#Main-Program">Program</a> </li>
+            <li> <a class="navbar-fun" href="#Featured">Featured Games</a> </li>
+            <li> <a class="navbar-fun" href="#Studio">Studio</a> </li>
+            <li> <a class="navbar-fun" href="https://www.youtube.com/watch?v=T5VtfD2i3rc">Watch Live</a></li>
+          </ul>
+        </nav>
+      </header>
```

Comment:
- [ ] As requested by the previous reviewer https://github.com/Hassaanjbaig-code/Capstone-Project/pull/1#discussion_r1031343860 Please make sure your header and Hero section is similar to the original design as much as possible currently your design is :point_down: 
![image](https://user-images.githubusercontent.com/97521557/203845207-4e089a42-3b09-45d4-9771-25b1f2f4268e.png)

I think you can use 100vh height on your main and remove the fixed height and width from your home-1 and add width 100% and align-items: center and use justify-content: space around or center for your header, this should fix the style for the design
```
main {
  background-position: 24px 57px;
  background-size: cover;
  height: 100vh;
}

.Home-1 {
  width: 100%;
  align-items: center;
}

header {
  width: 100%;
  background-color: #fff;
  justify-content: center;
}
```